static/debug.html

```diff
@@ -17,38 +17,10 @@
    (Angular Scenario, for example) need the body to be loaded so that it can insert its magic
    into it. If it is before body, then it fails to find the body and crashes and burns in an epic
    manner. -->
+  <script src="context.js"></script>
+  <script src="context-debug.js"></script>
   <script type="text/javascript">
-    window.__karma__ = {
-      info: function(info) {
-        if (info.dump && window.console) window.console.log(info.dump);
-      },
-      complete: function() {
-        if (window.console) window.console.log('Skipped ' + this.skipped + ' tests');
-      },
-      store: function() {},
-      skipped: 0,
-      result: window.console ? function(result) {
-        if (result.skipped) {
-          this.skipped++;
-          return;
-        }
-        var msg = result.success ? 'SUCCESS ' : 'FAILED ';
-        window.console.log(msg + result.suite.join(' ') + ' ' + result.description);
-
-        for (var i = 0; i < result.log.length; i++) {
-          //throwing error without loosing stack trace
-          (function (err) {
-            setTimeout(function() {
-              throw err;
-            });
-          })(result.log[i])
-        }
-      } : function() {},
-      loaded: function() {
-        this.start();
-      }
-    };
-
+    // Configure our Karma
     %CLIENT_CONFIG%
```

Comment:
We could inject `CLIENT_CONFIG` inside of the `.js` files but would need to do it in a linter friendly way. For example:

```js
// In client.js
window.__karma__.config = {PLACEHOLDER: 'FOR_CLIENT_CONFIG'}

// In middleware
data = data.replace('{PLACEHOLDER: \'FOR_CLIENT_CONFIG\'}', configJson);
```

imo, the current setup is good since it's minimal tax to maintain (e.g. no chance of escaping errors) and there are no major benefits to the other setup